src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -320,7 +320,7 @@ namespace System.Drawing
         private const int ARGBRedShift = 16;
         private const int ARGBGreenShift = 8;
         private const int ARGBBlueShift = 0;
-
+        private const uint ARGBAlphaMask = 0xffu << ARGBAlphaShift;
```

Comment:
We usually use capital letters in hex numbers.

Reply:
Ah, ok... this file had a mix of the two.  Will change